tox.ini

```diff
@@ -9,10 +9,16 @@ skip_install=True
 deps =
   pytest >= 4.6.3, < 5.0
   pyyaml >= 5.1.2, < 6.0
+  flake8 >= 3.7.9, < 3.8
 commands =
     pytest {posargs}
 
 [testenv:lint]
 skip_install=True
 commands = flake8 src doc data tests
 deps = flake8
+
+[testenv:invalidcode]
```

Comment:
idk how to call this tho, feel free to suggest a different name if you have better ideas